introduction.md

```diff
@@ -1 +1,9 @@
-Hello - my name is Eric Lamy. I'm a developer and project manager in Newburyport, MA. My favorite Code School path is HTML & CSS.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?